common/utils/src/main/resources/error/error-classes.json

```diff
@@ -3270,6 +3270,36 @@
     ],
     "sqlState" : "XXKST"
   },
+  "STV2_ENCODER_UNSUPPORTED_PREFIX_KEY" : {
+    "message" : [
+      "Encoder doesn't support prefix key for state store provider: <stateStoreEncoder>. Please use another state store provider that supports prefix keys, or omit the prefix key."
+    ],
+    "sqlState" : "00000"
+  },
+  "STV2_IMPLICIT_KEY_NOT_FOUND" : {
+    "message" : [
+      "Implicit key not found for operation on stateName=<stateName>"
+    ],
+    "sqlState" : "55019"
+  },
+  "STV2_STORE_MULTIPLE_VALUES_PER_KEY" : {
+    "message" : [
+      "Store does not support multiple values per key"
+    ],
+    "sqlState" : "42802"
+  },
+  "STV2_UNSUPPORTED_OPERATION" : {
+    "message" : [
+      "<operationType> operation not supported with <entity>"
+    ],
+    "sqlState" : "000A0"
+  },
+  "STV2_VALUE_SHOULD_BE_NONNULL" : {
```

Comment:
This error is for when the user tries to append a null value to a list, or set an existing list to a null value.